server/src/com/cloud/api/ApiDBUtils.java

```diff
@@ -559,6 +561,8 @@
     @Inject
     private VpcManager vpcMgr;
     @Inject
+    private AccountManager accountManager;
```

Comment:
Why there is a need to inject AccountManager? If the config key is a static it can be accessed as AccountManager.<static config key>